docs/devGuide/development/workflow.md

```diff
@@ -190,6 +190,20 @@ After which, you can update the **expected** test files with: `npm run updatetes
   as uncommitted changes due to the way they are generated. If you are not directly modifying those files in your PR, you should **discard those changes** and **do not commit** them.
 </box>
 
+<box type="tip" seamless header="Solving merge conflicts in expected test files">
+
+1. Ensure the fork is latest
+   - Pull from upstream repo to fork
+2. Ensure local master is in sync with fork master
+   - Pull from the fork master into local master
+3. Checkout to PR branch
+4. Merge local master into PR branch 
+   - `git merge master`
+5. Accept any change to conflicts in generated test files
```

Comment:
Perhaps we could include examples of files/filetypes of generated functional test files!

Reply:
Hi @itsyme, thanks for the comment! I added a tooltip explaining the location of these expected files above (`expected` for functional and `__snapshot__` for unit test)